yang/ietf-energy-saving-mgt.yang

```diff
@@ -228,8 +228,8 @@ module ietf-energy-saving-mgt {
     base energy-saving-power-state;
     description
       "Indicates that some components with energy management 
-       support are available and can take measures to use 
-       less energy.";
+       support are not available and these components can
```

Comment:
I'm having troubles with this. What components that are not available can take measures?

Reply:
@billwuqin, please double check the description. Thanks